index.html

```diff
@@ -417,12 +417,12 @@
       </div>
       <div>
         <pre class="idl">dictionary RTCIceCandidate {
-             DOMString              foundation;
-             unsigned long          priority;
-             DOMString              ip;
-             RTCIceProtocol         protocol;
-             unsigned short         port;
-             RTCIceCandidateType    type;
+             required DOMString              foundation;
+             required unsigned long          priority;
+             required DOMString              ip;
+             required RTCIceProtocol         protocol;
+             required unsigned short         port;
+             required RTCIceCandidateType    type;
              RTCIceTcpCandidateType tcpType;
              DOMString              relatedAddress = "";
              unsigned short         relatedPort;
```

Comment:
The same should probably be applied to `relatedPort` which doesn't need to be `0` by default if its optional (see its description).